LensGalleyPlugin.inc.php

```diff
@@ -199,12 +199,16 @@ function _getXMLContents($request, $galley) {
 		$contents = $fileService->fs->read($file->path);
 
 		// Replace media file references
-                $embeddableFilesIterator = Services::get('submissionFile')->getMany([
-                        'assocTypes' => [ASSOC_TYPE_SUBMISSION_FILE],
-                        'assocIds' => [$submissionFile->getId()],
-                        'fileStages' => [SubmissionFile::SUBMISSION_FILE_DEPENDENT],
-                        'includeDependentFiles' => true,
-		]);
+		$collector = Repo::submissionFiles()
+			->getCollector()
+			->filterByAssoc(
+				ASSOC_TYPE_SUBMISSION_FILE,
+				[$submissionFile->getId()]
+			)
+			->filterByFileStages([SubmissionFile::SUBMISSION_FILE_DEPENDENT])
+			->filterByIncludeDependentFiles(true);
+		$embeddableFilesIterator = Repo::submissionFiles()->getMany($collector);
```

Comment:
Since `$embeddableFilesIterator` is only used in `iterator_to_array` below, and `$embeddableFiles` is only used in the `foreach`, I'd suggest getting rid of the conversion to array and just using the results of `getMany` directly.